File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/page_plus_ai_model_price_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PagePlusAiModelPriceVO:
    total_pages: int = None
    total_elements: int = None
    size: int = None
    content: List[PlusAiModelPriceVO] = None
    number: int = None
    sort: SortObject = None
    first: bool = None
    last: bool = None
    number_of_elements: int = None
    pageable: PageableObject = None
    empty: bool = None

File: sdkwork-backend-sdk-typescript/src/types/page-plus-ai-model-price-vo.ts
import type { PageableObject } from './pageable-object';
import type { PlusAiModelPriceVO } from './plus-ai-model-price-vo';
import type { SortObject } from './sort-object';

export interface PagePlusAiModelPriceVO {
  totalPages?: number;
  totalElements?: number;
  size?: number;
  content?: PlusAiModelPriceVO[];
  number?: number;
  numberOfElements?: number;
  first?: boolean;
  last?: boolean;
  sort?: SortObject;
  pageable?: PageableObject;
  empty?: boolean;
}
